src/DssSpell.t.sol

```diff
@@ -36,6 +36,90 @@ contract DssSpellTest is DssSpellTestBase {
         checkCollateralValues(afterSpell);
     }
 
+    function giveTokensGUSD(DSTokenAbstract token, uint256 amount) internal {
```

Comment:
If it is a function specifc for GUSD, why are we looking for the storage slot and not just set it directly?

Reply:
I'm lazy? lol

Can set it direct if you really want. :)

Reply:
Haha no no, it's fine, let's leave this way.